client/db/bolt/db.go

```diff
@@ -298,9 +299,30 @@ func (db *BoltDB) CreateAccount(ai *dexdb.AccountInfo) error {
 	})
 }
 
+// Make account info key from client pub key
+func accountInfoKey(accountID account.AccountID) []byte {
+	acctKey := make([]byte, len(accountID))
+	copy(acctKey, accountID.String())
+	return acctKey
+}
+
+// DisableAccount sets account info disabled flag to true
```

Comment:
Please end docs with period, and with full sentences.  The exported docs end up shown in the package documentation in pkg.go.dev and elsewhere.